src/components/App.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import axios from 'axios';
+import './app.css'
+import MenuList from './MenuList'
+import { Header, Grid, Item, Icon } from 'semantic-ui-react'
+
+class App extends React.Component {
+  state = {
+    products: []
+  }
+
+  componentDidMount() {
+    this.getAllProducts()
+  }
+
+  getAllProducts = async () => {
+    let response = await axios.get('http://localhost:3000/api/products')
+    this.setState({ products: response.data.products })
+  }
```

Comment:
Move the logic regarding fetching the products to the MenuList component instead, it is not the responsibility of the App. We want to keep our app as clean as possible and only handle state here that concerns the children components.